firstservice/src/main/java/com/services/firstservice/service/UserService.java

```diff
@@ -0,0 +1,50 @@
+package com.services.firstservice.service;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.services.firstservice.db.entity.user.User;
+import com.services.firstservice.db.repositories.user.UserRepository;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class UserService {
+
+    private final UserRepository userRepository;
+    private final RestTemplate restTemplate;
+
+    public User get(Long id) {
+        return userRepository.findById(id).get();
+    }
+
+    public User create(User user) {
+        HttpEntity<User> entity = new HttpEntity<User>(user);
+        ResponseEntity<User> responseEntity = restTemplate.exchange("http://secondservice:8082/user", HttpMethod.POST,
```

Comment:
Move your secondservice address to `application.properties`, do not hard code it here. Same for the other services.